app/models/concerns/relationship_bulk.rb

```diff
@@ -29,6 +29,9 @@ def bulk_update(ids, updates, team)
         }
         self.delay.run_update_callbacks(ids.to_json, extra_options.to_json)
         delete_cached_fields(pm_source.id, relationships.map(&:target_id))
+        # Call the cached field sources_count to update its value, as the field is not called from the UI.
+        ids = [pm_source.id, relationships.map(&:target_id)].flatten
+        ProjectMedia.where(id: ids).map(&:sources_count)
```

Comment:
If you really want to update the value, shouldn't you call `sources_count(true)`?

Reply:
Cached already cleared before this line so both are same.